CatProducer/plugins/RecoEventInfoProducer.cc

```diff
@@ -120,11 +120,14 @@ void RecoEventInfoProducer::produce(edm::Event& event, const edm::EventSetup& ev
       const unsigned int trigIndex = hltConfig_.triggerIndex(trigName);
       if ( trigIndex < hltHandle->size() )
       {
-        if ( hltHandle->accept(trigIndex) ) { isPassed = true; break; }
+	if ( hltHandle->accept(trigIndex) ) {
+	  const std::pair<int,int> prescales = hltConfig_.prescaleValues(event, eventSetup, trigName);
+	  psValue = prescales.first * prescales.second;
+	  break;
```

Comment:
Actually I added break, not a problem without PS.
Combining 2 or more paths with PS'ed should be avoided.  
How about changing 
```cpp
psValue = prescales.first * prescales.second;
break;
```
to 
```cpp
psValue = min(psValue, prescales.first * prescales.second);
//break;
```
does this make sense?